### PR TITLE
apis: introduce cpu evict fields in NodeSLO

### PR DESCRIPTION
**Pull request by jasonliu747**
Signed-off-by: Jason Liu <jasonliu747@gmail.com>

### Ⅰ. Describe what this PR does
This PR introduces 
```
// cpu evict threshold percentage (0,100), default = 70
CPUEvictThresholdPercent *int64 `json:"cpuEvictThresholdPercent,omitempty"`

// if be CPU RealLimit/allocatedLimit > CPUEvictBESatisfactionUpperPercent, then stop evict bepods
CPUEvictBESatisfactionUpperPercent *int64 `json:"cpuEvictBESatisfactionUpperPercent,omitempty"`

// if be CPU (RealLimit/allocatedLimit < CPUEvictBESatisfactionLowerPercent and usage nearly 100%) continue CPUEvictTimeWindowSeconds,then start evict
CPUEvictBESatisfactionLowerPercent *int64 `json:"cpuEvictBESatisfactionUpperPercent,omitempty"`

// cpu evict start after continue avg(cpuusage) > CPUEvictThresholdPercent in seconds
CPUEvictTimeWindowSeconds *int64 `json:"cpuEvictTimeWindowSeconds,omitempty"`
```
 in NodeSLO API. It will later be used for CPU evict feature in Koordlet.

### CODE REVIEW

File: apis/slo/v1alpha1/nodeslo_types.go [hunk 1]
```diff
@@ -160,11 +159,22 @@ type ResourceThresholdStrategy struct {
...
 	// +kubebuilder:validation:Maximum=100
 	// +kubebuilder:validation:Minimum=0
 	MemoryEvictLowerPercent *int64 `json:"memoryEvictLowerPercent,omitempty"`
+
+	// cpu evict threshold percentage (0,100), default = 70
+	// +kubebuilder:default=70
+	// +kubebuilder:validation:Maximum=100
+	// +kubebuilder:validation:Minimum=0
+	CPUEvictThresholdPercent *int64 `json:"cpuEvictThresholdPercent,omitempty"`
+	// if be CPU RealLimit/allocatedLimit > CPUEvictBESatisfactionUpperPercent, then stop evict bepods
```
Comment (stormgbs) [inline apis/slo/v1alpha1/nodeslo_types.go:172]: BE pods

Comment (jasonliu747) [reply apis/slo/v1alpha1/nodeslo_types.go]: fixed

**Review by stormgbs — APPROVED** [review]

**Pull request merged**